Automattic/jetpack
# Nav Unification: Display the stats sparkline on WP Admin for Atomic sites

Fixes https://github.com/Automattic/wp-calypso/issues/56542

#### Changes proposed in this Pull Request:
Displays the stats sparkline in the WP Admin sidebar of Atomic sites, effectively achieving both Calypso/WP Admin and Simple/Atomic parity.

<img width="269" alt="Screen Shot 2021-11-05 at 15 40 12" src="https://user-images.githubusercontent.com/1233880/140528669-fec35b1c-c834-401c-bfcd-bccb65ee86f6.png">

Note that the Stats module has been updated to allow a new `masterbar` param in chart URLs, which produces a slightly different version of the chart suitable for the sidebar (see r124501-wpcom).

#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
- Install Jetpack Beta on an Atomic site and activate to the branch of this PR.
- Go to any WP Admin screen of your Atomic site.
- Make sure the sparkline chart is displayed right next to the Stats menu.
- Make sure the chart matches the one displayed on Calypso screens.
- Turn off the Stats module (you can do this on the Blog RC).
- Reload the WP Admin screen.
- Make sure the sparkline chart doesn't show up.
